pkgs/development/python-modules/durus/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "durus";
+  version = "4.2";
+
+  src = fetchFromGitHub {
```

Comment:
In general, I'd prefer to fetch the release tarballs from Pypi via `fetchPypi` unless there's a reason against it.

Reply:
done